multimedia/music/spotify/pspec.xml

```diff
@@ -34,6 +34,14 @@
     </Package>
 
     <History>
+	<Update release="45">
+            <Date>09-15-2018</Date>
+            <Version>1.0.89.313</Version>
+            <Comment>Update to 1.0.89.313</Comment>
+            <Name>Thomas Staudinger</Name>
+            <Email>Staudi.Kaos@gmail.com</Email>
+        </Update>
```

Comment:
Indentation is bork.

Reply:
Sorry about that. Just goes to show that I shouldn't commit stuff when dead-tired :smiley: